docs/content/Configuration/VPC/Connecting-with-a-VPC-AWS.mdx

```diff
@@ -15,57 +15,96 @@ required:
   to see if Cube Cloud VPC connectivity is available in your region.
 - **AWS VPC ID:** The ID of the VPC that Cube Cloud will connect to, for example
   `vpc-0099aazz`
-- **AWS VPC CIDR:** The CIDR block of the VPC that Cube Cloud will connect to,
-  for example `10.0.0.0/16`
+- **AWS VPC CIDR:** The [CIDR block][wiki-cidr-block] of the VPC that Cube Cloud
+  will connect to, for example `10.0.0.0/16`
+
+After receiving the information above, a Customer Success Manager will provide
+you with an **AWS account ID**, **region**, a **VPC ID** and a [**CIDR
+block**][wiki-cidr-block] used by Cube Cloud to connect to your VPC.
```

Comment:
Should we abstract out "CSM," "Support," and other references to teams/roles, if any, to something less specific like "the Cube (Cloud) team (representative)?" If anything changes, it would be inconvenient to fish for such names. However, I understand that "a CSM" is more specific and less confusing than "a rep."

Reply:
I used CSM because that's what seemed the easiest to understand for users, but happy to change it. Perhaps we could do a larger sweep through the docs to fix that?

Reply:
I don't wanna this to be a blocker, not very significant.